src/Transformations.jl

```diff
@@ -178,6 +178,86 @@ function transform!(tn::TensorNetwork, config::AntiDiagonalGauging)
     return tn
 end
 
+Base.@kwdef struct ColumnReduction <: Transformation
+    atol::Float64 = 1e-12
+    skip::Vector{Index} = Symbol[]
+end
+
+function transform!(tn::TensorNetwork, config::ColumnReduction)
+    skip_inds = isempty(config.skip) ? openinds(tn) : config.skip
+
+    for idx in keys(tn.tensors)
+        tensor = tn.tensors[idx]
```

Comment:
Why not just iterate over `tn.tensors`? `idx` is not used anywhere else.
```suggestion
    for tensor in tn.tensors
```

Reply:
Yes, I missed that!